coralproject/talk
# load more comments to moderate if available

## What does this PR do?

This is a little enhancement for the moderators. If you're in a queue, and you're approving/rejecting, and then you reach the end of the visible comments. If there were more comments available, you'd have to click the Load More button. With this PR, this all happens automatically.

## How do I test this PR?

- have some pre-moderated comments in the queue.
- in another tab, make another comment or two.
- now go back to the mod queue and take care of all the visible comments
- your new(er) comments should load automatically.
